### PR TITLE
Soda Experience first commit, splash screen, project intro and navigation

### LINKED ISSUE #699: [SODA Experience] Create the new SODA Experience dashboard scaffolding with basic code structure and basic dependencies
**Issue/Feature Description:**
This is for the new SODA Experience Dashboard which will provide the users an easy way to access the following for any project that is integrated into the dashboard:
- Project Description
- Use Cases & Features
- Architecture Diagram
- Link to User Guides
- Link to Developer Guides
- Video Demos
- Link to project Dashboard
- Install project 
- 
**Why this issue to fixed / feature is needed(give scenarios or use cases):**
The current SODA dashboard was developed as a demo client for the OpenSDS projects( SODA Terra and Strato projects )
With new projects being added to the SODA Framework and to make it easier for users who install or want to install SODA projects the SODA Experience will provide a single window for new users to get onboard quickly.
 
**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by prostil**
SODA Experience first commit, following features  implemented 

- Project scaffolding 
- Splash screen
- Project intro 
- Navigation to project details 

Fixes #699

### CODE REVIEW

**Comment by anvithks** [comment]
@prostil Could you please share some screenshots ?
It will help in review as the CI still needs to be setup for this.
Thanks.

**Comment by prostil** [comment]
image

**Review by anvithks — APPROVED** [review]
LGTM. Minor comments added.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "sodaexp",
+  "version": "0.0.0",
```
[the remark this answers is not in the document]
Comment (prostil) [reply package.json:3]: These I will update in the next PR, this is the base code

**Pull request merged**